cmd/evm/runner.go

```diff
@@ -82,24 +82,33 @@ type execStats struct {
 	GasUsed        uint64        `json:"gasUsed"`        // the amount of gas used during execution
 }
 
-func timedExec(bench bool, execFunc func() ([]byte, uint64, error)) ([]byte, execStats, error) {
+func timedExec(bench bool, execFunc func() ([]byte, uint64, error)) (output []byte, stats execStats, execErr error, benchErr error) {
```

Comment:
Instead of adding more returns... so we wanted to do `Fatalf`, which essentually stops everything. How about doing a `panic` instead of returning the `benchErr`? More or less the same as `b.Fatalf`?

Reply:
I just think it's getting a bit unwieldy, this is supposed to be a neat little helper

Reply:
I agree that a panic is fine here:  this error case indicates that something is wrong with our implementation.